lib/app/sequence_base.rb

```diff
@@ -786,7 +786,7 @@ def resolve_element_from_path(element, path)
 
         path_ary = path.split('.')
         cur_path_part = path_ary.shift.to_sym
-        return nil if el_as_array.none? { |el| el.send(cur_path_part).present? }
+        return nil if el_as_array.none? { |el| el.send(cur_path_part).present? || el.send(cur_path_part) == false }
```

Comment:
I think line 797 needs to be updated as well.

Reply:
Updated.  Hard to completely verify this additional change is working properly thoroughly, so double check my logic on that one please (even though it is seemingly simple).

Reply:
Yeah, I'm not 100% certain it needs to be updated, but I think this update looks good.